src/server/utils/gateway.ts

```diff
@@ -5,7 +5,7 @@ import {Headers as DebugHeaders, GatewayConfig, GatewayError} from '@gravity-ui/
 import {AppContext, AppError, NodeKit} from '@gravity-ui/nodekit';
 
 import {Feature, isEnabledServerFeature} from '../../shared';
-import {getAuthHeadersNone} from '../../shared/schema/gateway-utils';
```

Comment:
`getAuthHeadersNone` is not used now, delete it plz

Reply:
You also got caught as me :) what about private part ?